R/in-notin.R

```diff
@@ -8,6 +8,9 @@
 #' @param x vector or NULL
 #' @param y vector or NULL
 #'
+#' @details
+#' Falls back to base::`%in%` when x and y don't share a common type. This means that the behaviour of base::`%in%` is preserved (e.g. "1" %in% c(1, 2) is TRUE) but loses the speedup provided by vec_in.
```

Comment:
Can you split this into multiple lines? It will still read the same in the docs but it's a little bit easier to read in the code.

```
#' Falls back to base::`%in%` when x and y don't share a common type.
#' This means that the behaviour of base::`%in%` is preserved (e.g. "1" %in% c(1, 2) is TRUE)
#' but loses the speedup provided by vec_in.
```